tools/dart_tools/lib/runCommand.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io';
+
+Future<int> runCommand(String exe, List<String> arguments,
```

Comment:
Please use `command` rather than `exe` so that it's more platform-neutral; this could be useful for Linux and macOS as well.